fury/tests/test_material.py

```diff
@@ -122,178 +122,155 @@ def test_manifest_standard():
-        actual = ss[125, 125, :] / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-        actual = ss[125, 75, :] / 1000
-        desired = np.array([1, 0, 85]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-
-        scene.clear()  # Reset scene
-
-        # Special case: Contour from roi
-        data = np.zeros((50, 50, 50))
-        data[20:30, 25, 25] = 1.
-        data[25, 20:30, 25] = 1.
-        test_actor = actor.contour_from_roi(data, color=np.array([1, 0, 1]))
-        scene.add(test_actor)
-
-        window.record(scene, out_path=tmp_fname, size=(200, 200),
-                      reset_camera=True)
-        npt.assert_equal(os.path.exists(tmp_fname), True)
-        ss = load_image(tmp_fname)
-        actual = ss[90, 110, :] / 1000
-        desired = np.array([253, 0, 253]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-        actual = ss[90, 60, :] / 1000
-        desired = np.array([180, 0, 180]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-
-        material.manifest_standard(test_actor)
-        window.record(scene, out_path=tmp_fname, size=(200, 200),
-                      reset_camera=True)
-        npt.assert_equal(os.path.exists(tmp_fname), True)
-        ss = load_image(tmp_fname)
-        actual = ss[90, 110, :] / 1000
-        desired = np.array([253, 253, 253]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-        actual = ss[90, 60, :] / 1000
-        desired = np.array([180, 180, 180]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-
-        material.manifest_standard(test_actor, diffuse_color=(1, 0, 1))
-        window.record(scene, out_path=tmp_fname, size=(200, 200),
-                      reset_camera=True)
-        npt.assert_equal(os.path.exists(tmp_fname), True)
-        ss = load_image(tmp_fname)
-        actual = ss[90, 110, :] / 1000
-        desired = np.array([253, 0, 253]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-        actual = ss[90, 60, :] / 1000
-        desired = np.array([180, 0, 180]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
+    scene = window.Scene()  # Setup scene
+
+    test_actor = actor.box(center, directions=(1, 1, 1), colors=(0, 0, 1),
+                           scales=1)
+    scene.add(test_actor)
+
+    # scene.reset_camera()
+    # window.show(scene)
+    ss = window.snapshot(scene, size=(200, 200))
+    actual = ss[75, 100, :] / 1000
+    desired = np.array([0, 0, 170]) / 1000
+    npt.assert_array_almost_equal(actual, desired, decimal=2)
+    actual = ss[125, 125, :] / 1000
+    npt.assert_array_almost_equal(actual, desired, decimal=2)
+    actual = ss[125, 75, :] / 1000
+    desired = np.array([0, 0, 85]) / 1000
+    # TODO: check if camera affects this assert
+    # npt.assert_array_almost_equal(actual, desired, decimal=2)
```

Comment:
I can see many of the assertions of the standard material are disabled. Do you want me to look into this potential camera issue and make a PR to your branch?